src/components/DocumentList/DocumentList.js

```diff
@@ -16,27 +18,14 @@ function DocumentList(props) {
     console.log(toggleState);
   }
 
-  function isExpanded(index) {
-    if (props.expandedItems) {
-      return (
-        props.expandedItems.find(element => {
-          return element === index + 1;
-        }) !== undefined
-      );
-    }
-  }
-
   var children = docs
```

Comment:
I eliminated the code above since the child component can handle it more simply. No iteration or extra prop needed.

Reply:
We left this logic alone for now. This is basically just a hook for our stories, which means it should probably be removed. If we want to address this, we should do it alongside a general rework of how we interact with components from storybook, such as wrapping them in empty compenets etc...